cinema/urls.py

```diff
@@ -1,10 +1,41 @@
-from django.urls import path
+from django.urls import path, include
+from rest_framework import routers
 
-from cinema.views import movie_list, movie_detail
+from cinema.views import (
+    GenreList,
+    GenreDetail,
+    ActorList,
+    ActorDetail,
+    MovieViewSet,
+    CinemaHallViewSet
+)
+
+router = routers.DefaultRouter()
+router.register("movies", MovieViewSet)
+cinema_halls_list = CinemaHallViewSet.as_view(
+    actions={"get": "list",
+             "post": "create"
+             })
```

Comment:
no it was almost good before. You didn't need to move parenthesis

Reply:
move parentheses as it was before and curly bracket above

Reply:
hope i understand you correctly